src/components/Button/src/Button.vue

```diff
@@ -272,8 +411,13 @@ export default {
 		min-width: max-content;
 	}
 
+	&.size_expandable {
+		min-height: 48px;
+		padding: 5px 20px;
```

Comment:
i think all of our margins and paddings (and explicit heights and explicit widths) are multiples of 8 to maintain a consistent visual rhythm vertically & horizontally, with the only exception being 4px but even that one is kinda okay because it's half of 8. is it possible to make the `padding` here `4px 16px` or `8px 24px`? a `padding` of `5px 20px` is kinda odd.